bin/build.js

```diff
@@ -1,41 +1,46 @@
 /* eslint-disable no-console */
-import esbuild from 'esbuild';
+import * as esbuild from 'esbuild';
 
-const buildDirectory = 'dist';
-const production = process.env.NODE_ENV === 'production';
+// Config output
+const BUILD_DIRECTORY = 'dist';
+const PRODUCTION = process.env.NODE_ENV === 'production';
 
 // Config entrypoint files
-const entryPoints = ['src/index.ts'];
+const ENTRY_POINTS = ['src/index.ts'];
 
-/**
- * Default Settings
- * @type {esbuild.BuildOptions}
- */
-const defaultSettings = {
+// Config dev serving
+const LIVE_RELOAD = !PRODUCTION;
+const SERVE_PORT = 3000;
+
+// Create context
+const context = await esbuild.context({
   bundle: true,
-  outdir: buildDirectory,
-  minify: production,
-  sourcemap: !production,
-  target: production ? 'es2017' : 'esnext',
-  entryPoints,
-};
+  entryPoints: ENTRY_POINTS,
+  outdir: BUILD_DIRECTORY,
+  minify: PRODUCTION,
+  sourcemap: !PRODUCTION,
+  target: PRODUCTION ? 'es2019' : 'esnext',
+  inject: LIVE_RELOAD ? ['./bin/live-reload.js'] : undefined,
```

Comment:
Awesome ✅ only for development. https://esbuild.github.io/api/#live-reload

Reply:
that's right, the live reload script is just injected when building for development